src/components/profile/Profile.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Please kindly delete this comment line to have a clean and professional codebase 😁 🙃